lnd_multihoplocalchainclaim_test.go

```diff
@@ -0,0 +1,352 @@
+// +build rpctest
```

Comment:
How about `lnd_multi_hop_local_chain_claim_test.go ` instead of `lnd_multihoplocalchainclaim_test.go `? The former is far more readable IMO. Also applies to all the other files created in this commit.

Reply:
Renamed